packages/flame_lint/lib/analysis_options_with_dcm.yaml

```diff
@@ -11,5 +11,7 @@ dart_code_metrics:
     - avoid-top-level-members-in-tests:
         include:
           - 'test/**/*_test.dart'
+    - avoid-unnecessary-type-assertions:
+      ignore-mixins: true
```

Comment:
Should this line have a dash too?

Reply:
No that is a parameter

Reply:
Oh that makes sense, I totally got confused

Reply:
yeah yaml can get weird for lists of keys mixed with objects